docs/gladier/installation.rst

```diff
@@ -4,109 +4,14 @@ Installation
-
-You may notice that both have similar values, ``gladier.tools.hello_world.HelloWorld``.
-Each Gladier tool comes with its own built-in flow so it can easily be tested separately.
-Gladier Tools also contain their own ``required_input``, default ``flow_input``, and
-defined ``funcx_functions``.
-
-More on these components later. For now, let's run the flow!
-
-.. code-block:: python
-
-   hello_cli = HelloGladier()
-   flow = hello_cli.run_flow()
-
-The first line instantiates the client with default values, which means it will automatically
-trigger login for Globus Auth and register Flows and FuncX functions as needed, without asking.
-
-The second line, ``flow = hello_cli.run_flow()``, iterates over all defined tools and starts the flow. A bunch of things happen
-here, including:
-
-* Gathering input from each tool
-* Validating input for each tool
-* Triggering a login, if needed
-* Registering funcx functions (And re-registering, if they have changed)
-* Registering the ``flow_definition`` (And re-registering, if it has changed)
-* Starting the flow
-
-Now, the flow will run and complete. This is great, but it would be nice to get more details on
-whats happening. The GladierClient comes with some built-in functions to help with this:
-
-
-.. code-block:: python
-
-  hello_cli.progress(flow['action_id'])
-  details = hello_cli.get_details(flow['action_id'], 'HelloFuncXResult')
-  pprint(details)
-
-``hello_cli.progress()`` will periodically query the status of the flow until it finishes. It's
-a nice way to watch progression as the flow executes. Once the flow has finished,
-``hello_cli.get_details()`` will fetch output from the Globus Flow, so it can be displayed in a
-readable format.
-
-Running A Gladier Tool
-----------------------
-
-A full code snippet for running everything above is here:
-
-.. literalinclude:: gladier_example.py
-   :language: python
-
-Remember to replace ``<your_funcx_endpoint>`` with the UUID for your endpoint.
-You can use ``funcx-endpoint`` to list the endpoints running on your machine.
-If you need to setup a new funcx-endpoint, see the
-`FuncX-Endpoint first time setup <https://funcx.readthedocs.io/en/latest/endpoints.html#first-time-setup>`_
-
-Running the code above with a live FuncX endpoint will result in the following:
-
-.. code-block:: bash
-
-  $ python hello_world.py
-  Starting login with Globus Auth, press ^C to cancel.
-  [ACTIVE]: State HelloFuncX of type Action started
-  [ACTIVE]: State HelloFuncX of type Action started
-  [ACTIVE]: An Action is running
-  [ACTIVE]: An Action is running
-  {'action_id': '058b3cf7-f602-4b9c-8b29-661779f7c129',
-   'details': {'completion_t': '1623959367.248737',
-               'exception': None,
-               'result': 'hello world',
-               'status': 'SUCCEEDED',
-               'task_id': '058b3cf7-f602-4b9c-8b29-661779f7c129'},
-   'release_after': 'P30D',
-   'state_name': 'HelloFuncX',
-   'status': 'SUCCEEDED'}
```

Comment:
The "Hello World" docs were mostly stale and had to be rewritten. Newer docs are based around Tar and Transfer, and prioritize `@generate_flow_definition` over writing custom flows where possible.